File: Backend/model/Invoice.go
package model

import (
	"github.com/jinzhu/gorm"
)

type Invoice struct {
	gorm.Model `swaggerignore:"true"`
	Image      string `gorm:"not null"` //base64 string
	Name       string
	UserID     int `gorm:"not null"`
	Info       string
}

File: PAMPAW_kotlin/app/src/main/java/edu/ufp/pam/pampaw_kotlin/Database/Invoice.kt
package edu.ufp.pam.pampaw_kotlin.Database

import androidx.room.*

/**
 * A biblioteca de persistência Room oferece uma camada de abstração sobre o SQLite
 * para permitir um acesso robusto à BD.
 */
@Entity(
    tableName = "invoices",
    indices = [Index(value = ["invoiceID"])]
)
data class Invoice(
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "invoiceID") var invoiceID: Int?,
    @ColumnInfo(name = "invoiceName") var invoiceName: String? = "",
    @ColumnInfo(name = "userID") var userID: Int?,
    @ColumnInfo(name = "info") var info: String? = "",
    @ColumnInfo(name = "image") var image: String? = ""
) {
    constructor() : this(0, "", 0, "","")
}